toast-tk-web-agent/src/main/java/io/toast/tk/agent/ui/MainApp.java

```diff
@@ -11,246 +11,236 @@
 import java.io.File;
 import java.io.IOException;
 import java.io.InputStream;
-import java.util.Properties;
+import java.net.UnknownHostException;
+import java.util.Map;
 
 import javax.imageio.ImageIO;
 
-import org.apache.commons.io.FileUtils;
+import io.toast.tk.agent.ui.i18n.CommonMessages;
+import io.toast.tk.agent.ui.i18n.MainAppMessages;
+import io.toast.tk.agent.ui.provider.ConfigPanelProvider;
+import io.toast.tk.agent.ui.provider.PropertiesProvider;
+import io.toast.tk.agent.ui.verify.IPropertyVerifier;
+import io.toast.tk.core.annotation.ActionAdapter;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'io.toast.tk.core.annotation.ActionAdapter'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)